ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_virtual_topology.schema.yml

```diff
@@ -18,6 +18,10 @@ $defs:
       dscp:
         type: int
         $ref: "eos_cli_config_gen#keys/router_adaptive_virtual_topology/keys/policies/items/keys/matches/items/keys/dscp"
+      lowest_hop_count:
```

Comment:
Should we document that this key is only supported in cv_pathfinder?

Reply:
```suggestion
      lowest_hop_count:
        description: |-
          Prefer paths with lowest hop-count.
          Only applicable for `wan_mode: "cv-pathfinder"`
```

Reply:
> am I rendering it only for cv_pathfinder? :p

The point was to let the user know that :)

We have updated the schema description.